ipcontroller_config.py

```diff
@@ -39,12 +39,12 @@
 # c.IPControllerApp.log_level = 30
 
 # Set the working dir for the process.
-# c.IPControllerApp.work_dir = u'/home/cekees'
+# c.IPControllerApp.work_dir = u'/work1/home/cekees'
 
 # ssh url for engines to use when connecting to the Controller processes. It
 # should be of the form: [user@]server[:port]. The Controller's listening
 # addresses must be accessible from the ssh server
-# c.IPControllerApp.engine_ssh_server = u''
```

Comment:
how many engines? Does it work with a smaller number?

On Wed, May 6, 2015 at 3:03 PM, Chris Kees <notifications@github.com> wrote:

> In ipcontroller_config.py
> <https://github.com/erdc-cm/ipython_profile/pull/2#discussion_r29807169>:
>
> >
> >  # ssh url for engines to use when connecting to the Controller processes. It
> >  # should be of the form: [user@]server[:port]. The Controller's listening
> >  # addresses must be accessible from the ssh server
> > -# c.IPControllerApp.engine_ssh_server = u''
>
> OK, I verified that I can ping the front end nodes from the compute nodes
> one of the front end node's interfaces.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/erdc-cm/ipython_profile/pull/2/files#r29807169>.
>